static/js/components/ObservationPlanRequestForm.jsx

```diff
@@ -246,6 +256,11 @@ const ObservationPlanRequestForm = ({ gcnevent }) => {
   const [skymapInstrument, setSkymapInstrument] = useState(null);
   const [selectedFields, setSelectedFields] = useState([]);
 
+  const defaultAirmassTime = Date(
+    dayjs(gcnevent?.dateobs).format("YYYY-MM-DDTHH:mm:ssZ")
+  );
+  const [airmassTime, setAirmassTime] = useState(defaultAirmassTime);
+
```

Comment:
Something doesn't work well here. You want by default to be the airmass at the dateobs, but for me on the frontend it just shows me the current time. Also, it says it has to be in UTC, but it shows me the current time in my local. I think this could confuse users. Also, maybe its better if the default airmass time is in the middle of the start and end date selected in the form. It might be confusing to see the "observability" of tiles outside of the time frame showed in the form. It could get someone to think they cant observe when in fact they could.

Reply:
@Theodlz I understand some of what you mean... could you make what I think are the small mods and show me?